Goobi/src/org/goobi/production/properties/ProcessProperty.java

```diff
@@ -59,9 +59,12 @@
 	private boolean currentStepDuplicationAllowed = false;
 	
 	public ProcessProperty() {
+		this.container = new Integer(0);
```

Comment:
Introducing a known bad usage to change / fix it later is not a good way. We need no more bad usage which must be fixed sometime.